test/e2e/data/infrastructure-docker/cluster-template-md-remediation.yaml

```diff
@@ -57,7 +57,7 @@ metadata:
 spec:
   template:
     spec:
-      customImage: k8s-snap:dev
+      customImage: k8s-snap:dev-old
```

Comment:
again, just being curious, is there any particular reason that we used `dev-old` here and `dev-new` in the other non-upgrade templates (e.g. `cluster-template-kcp-remediation.yaml`)?

Reply:
Not really, let's use `dev-old` everywhere.